### ISSUE TITLE
fix(demo): add Intl polyfill

### ISSUE THREAD

**Issue by piyapan039285** [valor-software/ngx-bootstrap#2215]
In demo page : http://valor-software.github.io/ngx-bootstrap/, DatePicker sample is not shown up and IE10 console log produced an error below

```
ReferenceError: 'Intl' is undefined
   {
      [functions]: ,
      description: "'Intl' is undefined",
      message: "'Intl' is undefined",
      name: "ReferenceError",
      number: -2146823279,
      stack: "ReferenceError: 'Intl' is undefined
   at i (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:850:361)
   at Anonymous function (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:850:838)
   at p (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:850:889)
   at t.format (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:850:3234)
   at t.prototype.transform (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:331:625)
   at Anonymous function (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:1158:2819)
   at t.prototype.detectChangesInternal (http://valor-software.com/ngx-bootstrap/0.30a88ecc284237d0f89b.chunk.js:1:294309)
   at t.prototype.detectChanges (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd589704f01.bundle.js:605:5744)
   at t.prototype.detectChangesInNestedViews (http://valor-software.com/ngx-bootstrap/vendor.118b8a8bcdd"
   }
```

**Comment by valorkin** [comment]
I was sure I have added polyfill for it, hm

**Comment by valorkin** [comment]
BTW this issue has nothing to date picker,
when you are using date pipe, you need to add this Intl polyfill for IE and Safari

**Comment by IlyaSurmay** [comment]
Fixed in #2238.
If you're using date pipe, you need to add Intl polyfill.
`npm i --save intl`
then add following lines in `polyfills.ts` of your app.
```
import 'intl';
import 'intl/locale-data/jsonp/en';
```
